website/source/docs/runtime/environment.html.md

```diff
@@ -84,12 +88,12 @@ environment variables.
     <td>`VAULT_TOKEN`</td>
     <td>The task's Vault token. See [Vault Integration](/docs/vault-integration/index.html) for more details</td>
   </tr>
-  <tr>
-    <td>`NOMAD_ADDR_<task>_<label`></td>
-    <td>The allocated address, given as IP:Port, for each task, port label</td>
-  </tr>
 </table>
 
+Port labels and task names will have any dashes `-` in their names replaced by
```

Comment:
```
~> Port labels and task names will have any dashes `-` in their names replaced by
underscores `_` when they're used in environment variable names such as
`NOMAD_ADDR_<task>_<label>`.
```

This will make it a popped out bubble to draw attention. See yellow box: https://www.consul.io/docs/guides/outage.html

Up to you to include.